backend/webpack.worker.config.js

```diff
@@ -15,13 +15,14 @@ module.exports = {
   plugins: [
     // These are not used for being built, and they can't build properly, so we exclude them.
     new webpack.NormalModuleReplacementPlugin(
-      /(canvas|dockerode|ws)/,
+      /(dockerode)/,
```

Comment:
Why remove `ws` and `canvas` from here? I just want to be extra careful, because the original reason I added them in was that even though the build would pass, it would crash the lambda function once it was deployed there. So don't want to change this unless there's a good reason.

Reply:
having "ws" in there was the root cause of NoOpResourceLoader error. The path to the class contains "browser" and as such "ws". the classes in that folder will be replaced with mock.js when transpiled. Also, ws and canvas are needed by the jsdom. Can you point to which lambda function you are concern about?

Reply:
Ok, got it. Previously, when we deployed, it would cause the entire `crossfeed-stage-api` function to stop working. But it's possible that with these changes, the function would work. Let's merge this PR and in the worst case, if the function stops working, then we can revert.

Reply:
I assume you mean revert staging while we fix any issues.